webui-src/app/chat/chat.js

```diff
@@ -108,69 +185,180 @@ const ChatLobbyModel = {
             () => '{"id":{"type": 3,"lobby_id":' + m.route.param('lobby') + '}, "msg":' + JSON.stringify(msg) + '}'
         );
     },
+    selected(info, selName, defaultName) {
+        let currid = (ChatLobbyModel.currentLobby.lobby_id || {xstr64: m.route.param('lobby')}).xstr64
+        return ((info.lobby_id.xstr64 === currid) ? selName : '') + defaultName;
+    },
+    switchToEvent(info) {
+        return () => {
+            ChatLobbyModel.currentLobby = info;
+            m.route.set('/chat/:lobby', { lobby: info.lobby_id.xstr64 });
+            ChatLobbyModel.loadLobby(info.lobby_id.xstr64); // update
+        };
+    },
+    setupEvent(info) {
+        return () => {
+            m.route.set('/chat/:lobby/setup', { lobby: info.lobby_id.xstr64 });
+            ChatLobbyModel.loadPublicLobby(info.lobby_id.xstr64); // update
+        };
+    }
+}
+
+// ************************* views ****************************
+
+/**
+* Message displays a single Chat-Message<br>
+* currently removes formatting and in consequence inline links
+* msg: Message to Display
+*/
+const Message = () => {
+  let msg = null; // message to display
+  let text = ''; // extracted text to display
+  let datetime = ''; // date time to display
+  let username = ''; // username to display (later may be linked)
+  return {
+    oninit: vnode => {
+      console.info('chat Message',vnode);
+      msg = vnode.attrs;
+      datetime = new Date(msg.sendTime * 1000).toLocaleTimeString();
+      username = rs.userList.username(msg.lobby_peer_gxs_id);
+      text = msg.msg.replaceAll('<br/>','\n').replace(new RegExp('<style[^<]*</style>|<[^>]*>','gm'),'');
+      console.info('chat Text', text);
+    },
+    view: () => m('.message', m('span.datetime', datetime), m('span.username', username), m('span.messagetext', text))
+  };
 }
 
+
 const Lobby = () => {
   let info = {};
+  let tagname = '';
+  let onclick = e => {};
+  let lobbytagname = '';
   return {
-    oninit: (v) => info = v.attrs.info,
-    view: (v) => m( '.lobby.' + (ChatRoomsModel.subscribed(v.attrs.info) ? 'subscribed':'public'), {
-      key: v.attrs.info.lobby_id.xstr64,
-      onclick: e => {
-        if (ChatRoomsModel.subscribed(v.attrs.info)) {
-          m.route.set('/chat/:lobby', {
-            lobby: v.attrs.info.lobby_id.xstr64
-          });
-        }
-      },
+    oninit: (v) => {
+        info = v.attrs.info;
+        tagname = v.attrs.tagname;
+        onclick = v.attrs.onclick || (e => {});
+        lobbytagname= v.attrs.lobbytagname || 'h5.mainname';
+    },
+    view: v => {
+      return m(ChatLobbyModel.selected(info, '.selected-lobby', tagname), {
+      key: info.lobby_id.xstr64,
+      onclick: onclick,
     }, [
-      m('h5', info.lobby_name === '' ? '<unnamed>' : info.lobby_name),
-      m('p', info.lobby_topic),
-    ]),
+      m(lobbytagname,info.lobby_name === '' ? '<unnamed>' : info.lobby_name),
```

Comment:
Would be cleaner to have `h5` as the element type since it's not changing anyways, and pass class separately.
Like something this:
```js
m('h5', {class: lobbytagname}, ...
```